src/CodeReview.Orchestrator/Services/ContainerService.cs

```diff
@@ -23,6 +29,21 @@ public class ContainerService : IContainerService
             _nameFactory = nameFactory ?? throw new ArgumentNullException(nameof(nameFactory));
         }
 
+        private async Task ValidateImage(IDockerClient client, string imageName) {
+            var imageListParams = new ImagesListParameters {
+                MatchName = imageName
+            };
+            // Due to some reasons this method ignores match by image name and returns list of all registered images. Thus it's necessary to perform manual lookup.
+            var matches = await client.Images.ListImagesAsync(imageListParams);
+            var found = false;
+            if (matches != null && matches.Count > 0) {
```

Comment:
I would follow formatting approach used for rest of code. E.g. put `{` in separate line`

Reply:
done